### PR TITLE
reorganize script folders, move from doc/

### LINKED ISSUE #2929: reorganize folders/scripts in doc/
We place all kind of scripts and utilities in the doc/ folder. I think this is an unfortunate situation and I would like to move things around.
Suggestion:
./contrib/utilities/ for indent, update_prm_files.sh, update_source_files.sh
./contrib/release/ for update_copyright, release-tasklist, bump_version.sh
./contrib/docker/ for the docker files (from ./docker)

### PR DESCRIPTION
**Pull request by tjhei**
closes #2929

### CODE REVIEW

File: contrib/release/bump_version.sh [hunk 1]
```diff
@@ -13,12 +13,12 @@ VER=$1
...
 echo "manual.tex ..."
-sed -i "s/^.* %VERSION-INFO%\$/$VER %VERSION-INFO%/g" manual/manual.tex 
+sed -i "s/^.* %VERSION-INFO%\$/$VER %VERSION-INFO%/g" ../../doc/manual/manual.tex
 
 echo "global.h ..."
-sed -i "s|version \(.*\)\\\\n\" //VERSION-INFO|version $VER\\\\n\" //VERSION-INFO|g" ../include/aspect/global.h
+sed -i "s|version \(.*\)\\\\n\" //VERSION-INFO|version $VER\\\\n\" //VERSION-INFO|g" ../../include/aspect/global.h
 
-git add ../VERSION manual/manual.tex ../include/aspect/global.h
+git add ../VERSION ../../doc/manual/manual.tex ../../include/aspect/global.h
```
Comment (gassmoeller) [inline contrib/release/bump_version.sh:24]:
```suggestion
git add ../../VERSION ../../doc/manual/manual.tex ../../include/aspect/global.h
```

**Comment by tjhei** [comment]
> I am a bit worried that there might be other paths that would need adjusting, but I guess there is no way to test that before the next release.

Well, I am always going through the steps carefully and I will adjust things then.

**Comment by tjhei** [comment]
Fixed.

**Comment by gassmoeller** [comment]
The two update_script tests fail because the path is hardcoded inside the tests .cc file.

**Comment by tjhei** [comment]
> The two update_script tests fail because the path is hardcoded inside the tests .cc file.

Will fix this, but I might do this during the demo tomorrow. :-)

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -10,6 +10,8 @@
 \usepackage{gensymb}
 \usepackage{siunitx}
 
+
+
```
Comment (tjhei) [inline doc/manual/manual.tex:14]: this shouldn't be here. :-)

**Comment by gassmoeller** [comment]
Tests are still failing

**Comment by tjhei** [comment]
not anymore. :-)

**Pull request merged**